src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/ComparisonResults.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@model IEnumerable<Scheme>
 
 @section BreadCrumbs {
-    <a href="/" class="govuk-back-link">Home</a>
+    <a href="javascript:history.back()" class="govuk-back-link">Back</a>
```

Comment:
Can we use asp-route, as this needs to work without js.

Reply:
OK, changed back to /scheme-comparison